### DISCUSSION TITLE
error correction output

### DISCUSSION THREAD

**Discussion by INFINITY1993** [ablab/spades#967] [Q&A]
Hello,
For the BayesHammer output during metaspades assembly, there are three files in the corrected folder, namely *__unpaired.00.0_0.cor.fastq.gz, *_1.00.0_0.cor.fastq.gz and *_2.00.0_0.cor.fastq.gz. I have three questions on this issue: 
(1) what does the *__unpaired.00.0_0.cor.fastq.gz file mean? unpaired? 
(2) I also noticed that the total reads of the three files are less than the paired reads before error correction, so does BayesHammer remove a part of reads during the error correction? 
(3) For binning, how should I take the corrected reads as input? as you know, the binners, such as metabat2, need paired reads files as input?
Thanks so much!
Best regards!
![1650892988(1)](https://user-images.githubusercontent.com/63569349/165097833-75a90d81-2fc2-4f97-b6a2-7f565f5a92de.png)

**Comment by asl** [comment]
> (1) what does the *__unpaired.00.0_0.cor.fastq.gz file mean? unpaired?

Yes

> (2) I also noticed that the total reads of the three files are less than the paired reads before error correction, so does BayesHammer remove a part of reads during the error correction?

Yes. It might happen that one read in the pair is completely bad. Then it is removed and its mate might appear to be unpaired.

> (3) For binning, how should I take the corrected reads as input? as you know, the binners, such as metabat2, need paired reads files as input?

We would suggest you to take original reads in such case.
